simpleLooping/countPositiveAndNegativeInteger.py

```diff
@@ -3,3 +3,24 @@
 # Ex:
 #   countPositiveAndNegativeInteger([-9, -5, -2, -6, -3, -1]) # return (0, 6)
 #   countPositiveAndNegativeInteger([3, 5, 7, -2, 6, -1, -3]) # return (4, 3)
+from typing import List
+lst = ([-9, -5, -2, -6, -3, -1])
```

Comment:
Why do we have this lists here? This is unsafe because it might expose our program/project to security bug.